### DISCUSSION TITLE
BT-01(c)-Procedure incorrect example?

### DISCUSSION THREAD

**Discussion by duncandewhurst** [OP-TED/eForms-SDK#1064] [Q&A] [answered]
According to the eForms documentation for [BT-01-notice](https://docs.ted.europa.eu/eforms/latest/schema/notice-information.html#legalBasisSection), the EU regulatory domain that applies to the procurement should be identified by its CELEX number in `<cbc:RegulatoryDomain>` and 'national or other legal bases' should be identified in BT-01(c)-Procedure:

![image](https://github.com/user-attachments/assets/462fd74b-bf25-48e3-9583-ada458f3fc80)

Likewise, the documentation for [BT-01(c)-Procedure](https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html#otherLegalBasisSection) suggests that it should only be used for a procedure's 'local legal basis':

![image](https://github.com/user-attachments/assets/64791757-5108-49c7-9f3d-e53c83d5bf13)

However, the example given in the eForms documentation for BT-01(c)-Procedure is the ELI for Directive 2014/24, i.e. the EU regulatory domain, not the local legal basis:

![image](https://github.com/user-attachments/assets/13890e8d-bc5d-46f0-a5d2-a409a6f871cf)

Should the example instead show the identifier for the directive's transposition into local law? For example, in the UK (pre-brexit) the Public Contracts Regulations 2015 (ELI: https://www.legislation.gov.uk/id/uksi/2015/102) was the act that implemented the EU directive.

Also, according to the [identifiers documentation](https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html), when the identifier published in BT-01(c)-Procedure is an ELI, the `schemeName` attribute of  `<cbc:ID>` should be set to "ELI". However, that attribute is missing from the example in the documentation.

FYI @bjornjorgensen - this is a follow up to our email discussion.

**Comment by YvesJo** [comment, marked as answer]
Hi,
The documentation certainly needs some update like the addition of the schemeName attribute and some additional clarification on the use of the field.
The statements are correct and it was initially foreseen to have ELI, however not all references have an ELI identifier and not all available ID formats are known, therefore it has been decided to not apply controls on the specified values so that some flexibility is left to the author.
KR

**Comment by duncandewhurst** [comment]
Thanks, @YvesJo. To confirm: when you say "the statements are correct", do you mean that BT-01(c)-Procedure should not be used to reference EU directives (by ELI or otherwise)?

**Reply by YvesJo** [reply]
Hi BT-01(c)-Procedure is not there to replace BT-01-Procedure. BT-01-procedure is used to specify the directive that applies to the procedure (e.g. D24). BT-01(c)-Procedure is available to specify the ID of any additional legislation of interest for the procedure (except the ones that already have dedicated fields like Cross Boarder Law). I do not see any case where the EU directives should be specified in BT-01(c)-Procedure and not in BT-01-Procedure. If such a case would exist and would be justified, the current rules do not prohibit this.

**Reply by jpmckinney** [reply]
@YvesJo Does it make sense for BT-01(c)-Procedure to repeat the same Directive as BT-01-notice (for example, `<cac:ProcurementLegislationDocumentReference><cbc:ID>http://data.europa.eu/eli/dir/2014/24/oj</cbc:ID></cac:ProcurementLegislationDocumentReference>` and `<cbc:RegulatoryDomain>32014L0024</cbc:RegulatoryDomain>`, which both refer to Directive 2014/24), or is it recommended to simply omit BT-01(c)-Procedure in that case?

Presently, the examples in the documentation refers to Directive 2014/24 from both fields, which seems strange.

**Reply by YvesJo** [reply]
@jpmckinney, this indeed does not make much sense.
Also keep in mind that the documentation is not based on a single case as this wouldn't  allow to illustrate all the situations.
For concrete complete examples it is better to refer to the provided notice examples.
The use of the general directive in BT-01(c)-Procedure originally made sense when the use of "other" was initially restricted to below threshold notices as thresholds are directives dependent and having "other" in BT-01-Procedure required to additionally have the associated directive identified in BT-01(c)-Procedure The extended forms could be used for below threshold notices for whatever directive.
I hope this answers your question

**Reply by duncandewhurst** [reply]
@YvesJo did you mean to refer to having "other" in BT-01-Notice (not BT-01-Procedure)?

In which case, is it correct that BT-01(c)-Procedure should only reference a Directive when BT-01-Notice is set to "other"? i.e. there shouldn't be any data in which both BT-01-Notice and BT-01(c)-Procedure both reference the same Directive?

**Reply by YvesJo** [reply]
Hi,
Indeed, having the same Legislation in both fields is not relevant. There is no restriction in the use of BT-01(c)-Procedure and the field could be used to refer to any legislation having an impact on the procedure and not covered by other fields like the ones for BT-01-Notice, cross border law, environmental, employment and fiscal legislation.

**Reply by duncandewhurst** [reply]
@YvesJo Thanks for clarifying. So, the semantics of BT-01(c)-Procedure can be either of the following:

1. A reference to the local legal basis of the procedure (i.e. the laws and regulations that govern the contracting process and grant legal authority to the procuring entity). For example:
  * In an above-threshold procedure, the transposition of an EU directive into national law.
  * In a below-threshold procedure, a national law governing below-threshold procedures.
2. A reference to any legislation (local or international) that impacts the procedure without directly governing it or granting legal authority. For example, in a below-threshold procedure, a reference to an EU directive that defines applicability thresholds, even though the procedure’s value falls below those thresholds.

Is that correct?

**Reply by YvesJo** [reply]
Hi,
In my view this is a quite fair statement. 
Technically spoken there aren't restrictions, Business wise, a Business confirmation would be better. I'll see who should preferably do this.
